nested/nfs-ha-vm.json

```diff
@@ -221,7 +221,7 @@
             "resources": [
                 {
                     "type": "extensions",
-                    "apiVersion": "2017-03-30",
+                    "apiVersion": "latest",
```

Comment:
Let's not use "latest" for apiVersion, let's have a unique date string around this commit/time-frame for every applicable mention of apiVersion.